### ISSUE TITLE
Unknown error 0 when out of quota

### ISSUE THREAD

**Issue by magnush0lm** [jotta/jotta-cli-issues#122]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):**
jottad executable : /usr/bin/jottad
jottad appdata    : /var/lib/jottad
jottad version    : 0.7.35160
jotta-cli version : 0.7.35160

**Description of problem:**
When you try to upload files but have or are about to exceed you quota, you get an unknown error:
ERROR  unknown error
XID: Z0JwbmJQYmdITUxV
        Error 0: UNKNOWN

**Expected:**
Error n: out of quota, quota exceeded or some other exception text that makes sense and points you in direction of a solution

**jotta-cli status (`jotta-cli status`):**
```
 Account   : test-users@domain.com on Jottacloud
 Usage     : 4.88GiB / 5.00GiB
 Device    : jotta-test
------------------------------------------------------------------------------
 No folders added to backup yet

```

**Relevant logs for the issue (`cat ~/.jottad/jottabackup.log`)**
```
cat: /home/debian/.jottad/jottabackup.log: No such file or directory

```

**Traceback**
```

```

**Additional info:**

sudo apt-get install wget apt-transport-https ca-certificates

# this is missing in you howto at https://docs.jottacloud.com/en/articles/1436855-jottacloud-cli-for-linux-debian-packages
sudo apt-get install gnupg2

echo "deb https://repo.jotta.us/debian debian main" | sudo tee /etc/apt/sources.list.d/jotta-cli.list
wget -O - https://repo.jotta.us/public.gpg | sudo apt-key add -
sudo apt-get install jotta-cli
jotta-cli login

# log in with fresh, free account

# create dummy file
truncate -s 1000M 1000m.dump

# upload it until you reach quota
jotta-cli archive 1000m.dump --remote=/Archive/1000m1.dump
jotta-cli archive 1000m.dump --remote=/Archive/1000m2.dump
jotta-cli archive 1000m.dump --remote=/Archive/1000m3.dump
jotta-cli archive 1000m.dump --remote=/Archive/1000m4.dump
jotta-cli archive 1000m.dump --remote=/Archive/1000m5.dump
jotta-cli archive 1000m.dump --remote=/Archive/1000m6.dump

ERROR  unknown error
XID: Z0JwbmJQYmdITUxV
        Error 0: UNKNOWN

**Comment by coopersimon** [comment]
Thanks for the issue! We will investigate and aim to fix this in the next release.

**Comment by Kimbsen** [comment]
We released 0.8.36055 today which should provide a clearer error message for this and other errors.
